apple/internal/rule_support.bzl

```diff
@@ -110,6 +115,7 @@ def _describe_rule_type(
             that should hold the icon sets (e.g. .xcassets or .xcstickers).
         archive_extension: Extension for the archive output of the rule.
         binary_infoplist: Whether the Info.plist output should be in binary form.
+        bundle_package_type: Four-character code representing the bundle type.
```

Comment:
This is set for everything but `macos_command_line_application` and `macos_dylib` since they are not bundles